Small batch of icon additions

hefaestos opened the pull request:
# Description
<!-- Please provide a short summary of what icons you added, changed, or linked
-->

Added and linked icons for: AlcoDroid, Carrot, Chwazi, Dice, Star Walk 2, Two Dots

## Icons addition information
<!-- Please specify if you added an icon that was requested in the icon request form, as seen below -->
### Icons added
* AlcoDroid (`org.M.alcodroid`)
* CARROT (`com.grailr.carrotweather`)
* Chwazi (`com.tendadigital.chwaziApp`)
* Dice (`fr.sevenpixels.dice`)
* Star Walk 2 (`com.vitotechnology.StarWalk2Free`)
* Two Dots (`com.weplaydots.twodotsanroid`)

### Icons linked
* AlcoDroid (linked `org.M.alcodroid` to `alcodroid.svg`)
* CARROT (linked `com.grailr.carrotweather` to `carrot.svg`)
* Chwazi (linked `com.tendadigital.chwaziApp` to `chwazi.svg`)
* Dice (linked `fr.sevenpixels.dice` to `dice.svg`)
* Star Walk 2 (linked `com.vitotechnology.StarWalk2Free` to `star_walk_2.svg`)
* Two Dots (linked `com.weplaydots.twodotsanroid` to `two_dots.svg`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](./github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

x9136 commented on svgs/daylio.svg:
What is the reason an upgrade is required?

hefaestos replied:
I had the art done before I saw the icon had been recently added, and included it because it looked to me like a (minor) improvement. (Wouldn't have bothered otherwise, and to be clear, had several other inadvertent dupes which I did not include because they weren't improvements.)

1./3. To my thinking, the matching the original mouth shape is preferable to making the mouth concentric with the outer circle, since my understanding of the goal was to re-render the original (to the extent feasible). Given that concentricity was also an option to the original icon artist, it seems that non-concentricity was a deliberate design choice.

2. The mouth/border space is 9px, which is sufficient to show as clearly delineated on my screen... if there's an explicit guideline there in the definition document that I missed, please let me know.

x9136 replied:
Ok. We'll probably come back to this later, so let's talk about how you've been able to apply the guides.

x9136 replied:
<img width="643" alt="Screenshot 2023-09-11 at 09 23 24" src="https://github.com/LawnchairLauncher/lawnicons/assets/60105060/f1731699-2705-4845-bd35-83c908e5ccb2">

There should be a stroke instead of a fill. It may be worth raising the mouth and eyes higher so that the lower part does not get dirty on the phone screen.

x9136 commented on svgs/alcodroid.svg:
<img width="634" alt="Screenshot 2023-09-11 at 09 00 22" src="https://github.com/LawnchairLauncher/lawnicons/assets/60105060/d45367a9-dcc7-4904-a868-076244a3c1b9">

Please use a stroke instead of a fill and make sure the size is 160×160px (or 160×*px, *×160px).

hefaestos replied:
Appreciate the detailed comments. What program are you using in the screenshots?

No problem fixing the size - I misinterpreted the guide there.

Stroke vs fill - there are quite a number of released icons that appear to me to use fills for shapes that could be strokes (eg, AntennaPod, Feedly, Google Translate)... is there a particular rationale underlying this?

x9136 replied:
> What program are you using in the screenshots?

A screenshot program built into the MacOS system. The screenshot is taken from the Figma window.

> is there a particular rationale underlying this?

I think at first there were either no [Lawnicons guides](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md#artboard--grid) or reviewers were taking all the icons for the sake of quantity. If you don't take the requirements of Lawnicons guides, then making icons with a stroke is better, because it's easier to edit them and check them for consistency with the overall style.

The icons in red in the picture are the ones that still have fill.
<img width="471" alt="Screenshot 2023-09-11 at 20 47 50" src="https://github.com/LawnchairLauncher/lawnicons/assets/60105060/b7f07995-856c-492b-9b57-86035c09b3c5">

hefaestos replied:
Ah, what rotten luck! 6 of the 8 icons I pulled for my comparison set - all apps on my own homescreen - are in red. They looked like they'd been touched recently, so I assumed they represented current good practice. I actually converted my icons *to* fills based on these being in the majority. XD Thanks!

x9136 replied:
![Screenshot 2023-10-07 at 02 06 32](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/3655467b-1485-4f49-801b-cd2d3f7908d6)

Please use only black (#000000) color everywhere.

x9136 commented on svgs/carrot.svg:
<img width="646" alt="Screenshot 2023-09-11 at 09 03 37" src="https://github.com/LawnchairLauncher/lawnicons/assets/60105060/81ec4bb8-8ac1-40dc-9bcd-7b8f0fcc7f51">

You need a stroke instead of a fill and a size of 160×160px (or 160×*px, *×160px). Don't forget to round off the end caps at the arrow and lightning bolt.

x9136 replied:
![Screenshot 2023-10-07 at 02 09 07](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/fef7881f-fc66-4c3c-9573-b07be57cd8d9)
Please use strokes and black (#000000) everywhere.

![Screenshot 2023-10-07 at 02 20 33](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/fbc27f3d-932a-48fb-aac8-4f154239a64e)
One option of what the end result might look like.

x9136 commented on svgs/star_walk_2.svg:
<img width="649" alt="Screenshot 2023-09-11 at 09 11 43" src="https://github.com/LawnchairLauncher/lawnicons/assets/60105060/00d3e4d5-f0ab-4e5e-8ff4-9cab69fbe160">

It's worth using a stroke instead of a fill. Remember to keep the size to 160×160px (or 160×*px, *×160px). Also be more careful with the composition center, now the icon has fallen down.

hefaestos replied:
Is centering the icon a strict rule, matter of taste, something else? I had intentionally offset the icon down to give a better visual impression of centering due to the asymmetrical shape - it fits pretty evenly within the circular guidemark.

hefaestos replied:
Also re: the daylio icon... realize it may just be a matter of taste, but is there a guideline/reference here? As is, it doesn't look out of place to me relative to other released icons. I'll fiddle around with it, just would like to get it right in one update rather than 5, ha.

![circle comparison](https://github.com/LawnchairLauncher/lawnicons/assets/144565696/fda38afa-c35f-46e7-a3a0-5b1e3cba5cfc)

x9136 replied:
> Is centering the icon a strict rule, matter of taste, something else?

It's not strictly, it depends on the shape. But almost all icons in the pack are centered, so a deliberately offset icon will look out of place among the others.

> Also re: the daylio icon... realize it may just be a matter of taste, but is there a guideline/reference here?

This is not taken into account in the guides, but in practice, icons with small indentation between lines (less than 16-18px with a stroke thickness of 12px) get visually dirty.

The icon will not be as huge as you see it here — on the phone screen it will be small, so the indentation will be reduced too. The result is a situation when two lines tend to stick together. That's why it gets dirty. If you want, you can download the debug apk (once you pass the style test) and check it for yourself.

x9136 replied:
![Screenshot 2023-10-07 at 02 24 40](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/f5f51880-f310-43a8-aee5-e2d162021393)

Please use strokes everywhere. In this case, it is sufficient to remove the fill and round the end cap.

x9136 replied:
![Screenshot 2023-10-15 at 08 52 32](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/f0113a97-146d-4650-8fdf-a3f9743d2f4d)
Please reduce the size so that the width is 160px.

hefaestos replied:
Hmm... So, when I open it, it shows as 160 to me...

![2023-10-15 17_00_57-star_walk_2 svg - Inkscape](https://github.com/LawnchairLauncher/lawnicons/assets/144565696/6d59bf51-3835-4353-a0d4-445300d1e275)

Also, previous examples you've shown for size have had the size box snug against the features (see below), but in this example the blue box extends beyond the features a bit - it's an equal 3.66 bigger on H and W than what I see. 

![266923790-00d3e4d5-f0ab-4e5e-8ff4-9cab69fbe160](https://github.com/LawnchairLauncher/lawnicons/assets/144565696/ee99fa6a-796d-4cbc-9723-8274a7ea5484)

Are you detecting an unshown feature, or... just trying to figure out what's going on here.

x9136 replied:
@hefaestos My bad. I didn't expect ellipses to have a rotation transformation — in this case Figma counts dimensions differently. I focused only on the numbers, visually everything is as it should be.

x9136 commented on svgs/two_dots.svg:
<img width="640" alt="Screenshot 2023-09-11 at 09 16 04" src="https://github.com/LawnchairLauncher/lawnicons/assets/60105060/4f5e27df-d628-4fd2-93a6-8408cea24066">

Keep in mind that the size should be 160×160px.

Perhaps the thickness of the stroke should be 12px. You can only say exactly by comparing on your phone among the other icons, so decide for yourself how much time to invest.

x9136 reviewed (commented):
Please replace the fill with a stroke and correct the dimensions so the icons match [the guides.](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md#artboard--grid)

x9136 commented on app/assets/appfilter.xml:
<img width="701" alt="Screenshot 2023-09-11 at 09 28 53" src="https://github.com/LawnchairLauncher/lawnicons/assets/60105060/f1bdbf07-9816-4fe6-8d99-3d8aad7a787e">

I think the spaces in the newlines are preventing it from passing the style check.

hefaestos replied:
Also thanks for this! Couldn't figure out if there was a catch in the actual code, or if I was just executing the process incorrectly.

hefaestos commented:
(not abandoned, just out of town for a week; will fix when I get home!)

hefaestos commented:
Amended icons as requested. Removed Daylio, added Chwazi and Dice.

x9136 commented on svgs/chwazi.svg:
![Screenshot 2023-10-07 at 02 30 57](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/293aa1c3-879a-4689-a158-8bc169e9ee07)
Please use strokes and black (#000000) everywhere.

![Screenshot 2023-10-07 at 02 37 29](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/b6641752-61e7-4df7-a880-e967c3a51c8a)
For stroke thickness, use only the values from [the guides](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md#foundation).



![Screenshot 2023-10-07 at 02 32 05](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/f1075427-fa4b-4ab5-8d83-9a9dd87f0861)
Make sure the size is 160×160px (or 160×*px, *×160px).

![Screenshot 2023-10-07 at 02 35 53](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/04fc4601-6620-487e-90bf-cca2df87a89e)
What this icon might look like.

hefaestos replied:
The square formed by the dots is centered in the 160x160 area. The intent is for one of the dots to be noticeably larger than the others, so as long as the centers are arranged in a square, it's going to wind up with some white space around the other dots. Is there just no way to do "square of circles, with one bigger" compliant to the guide?

x9136 replied:
> Is there just no way to do "square of circles, with one bigger" compliant to the guide?

There are forms that are difficult or impossible to do by guide, but this is what keeps the overall style in icons.

> I would like to understand what the point of noncompliance here is.

If the dimensions are incorrect, the icon will scale differently on the phone screen. As a result, it will be smaller than it should be among the others.

![Screenshot 2023-10-07 at 03 28 44](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/0ad15cdb-775e-4749-b679-9d8d6a4aa003)
If you want to make one circle larger than the others and position them relative to each other's centers, this is possible with an extra square. I prefer to keep it simple and do without squares.

hefaestos replied:
Sorry, the "understand what the point of noncompliance here is" was in reference to the Carrot Weather icon - you said to use strokes, but I believe I have? All of the shapes are single line... what else is incorrect other than the fills (which I thought I removed before saving off; sorry!).

With regards to the Chwazi one - I'll have to think on the best way to fix this... The native icon would have one a different color (eg, filled) which is also bad.  ¯\_(ツ)_/¯

x9136 replied:
> the Carrot Weather icon

I'm just now realizing which icon we're talking about. Each icon has its own topic, so it's unusual to talk about several icons in the comments of one of the icons.

In this icon, the fill is present in the arrow and in the lightning bolt. I've shown you how these shapes usually look like if you use a stroke.

I also simplified the icon because I talked about it in the last review. Too much detail usually ruins the look and doesn't look good on a phone screen.

hefaestos replied:
Yeh, sorry for the confusion; the page literally does not give me the option to reply to any of this batch of posts besides this one and Dice. I don't know what the issue is... (image attached).  I will rework these; thanks for your patience.

![no reply](https://github.com/LawnchairLauncher/lawnicons/assets/144565696/41ea835c-73c0-4209-8ef4-4cae695b02b3)

hefaestos replied:
thank you, that works. =|  (why they gotta hide it?)

x9136 commented on svgs/dice.svg:
![Screenshot 2023-10-07 at 02 42 18](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/b62718dd-037b-4e5f-9ed4-3bc5fbb217e7)
Please remove the fill, then correct the filled elements a bit afterwards.

x9136 commented on app/assets/appfilter.xml.bak:
It's worth deleting this file to keep it out of the repository.

hefaestos commented:
Apologies for the janky history - I screwed things up in a new and innovative way that my partner couldn't untangle, so had to wipe.

Fixed requested icons (color fix, fill fix, simplification, etc). Fixed a previous typo in one of my own icon calls, and fixed the Feedly call (looks like Feedly changed, because it used to work - "Main" to "MainActivity").

Installed it myself; I think they look ok...
![screenshot - installed icons](https://github.com/LawnchairLauncher/lawnicons/assets/144565696/baefeaea-194a-4138-8a6b-4b14bb251fac)
